lte/gateway/c/session_manager/LocalEnforcer.cpp

```diff
@@ -1078,6 +1078,63 @@ void LocalEnforcer::schedule_session_init_bearer_creations(
       LocalEnforcer::BEARER_CREATION_DELAY_ON_SESSION_INIT);
 }
 
+void LocalEnforcer::initialize_creating_session(
```

Comment:
isnt this the same as `init_session_credit`

Reply:
Haha yes I accidentally left this as not in draft but it is still in progress. It will essentially replace `init_session_credit`. So the upcoming change is to create a session with CREATING state when we receive the request from MME. Once we process the CreateSessionResponse from FeG we will transition to CREATED and add rules etc. But this is still in testing so didn't mean to publish it :D